reference/commands/graph/info.rst

```diff
@@ -230,25 +230,36 @@ For example, to get the options of zlib, the following command could be run:
         shared: False
 
 
-You can generate a graph of your dependencies in ``json``, ``dot`` or ``html`` formats:
+Available formatters
+--------------------
 
-Now, let's try the ``dot`` format for instance:
+json formatter
+^^^^^^^^^^^^^^
+
+For the documentation about the JSON formatter, please check the :ref:`dedicated section <reference_commands_graph_info_json_format>`.
```

Comment:
Maybe, it's the moment to bring that piece of documentation here. Wdyt?

Reply:
If I recall correctly I think we moved it there because there are several subcommands that use the same json formatter?

Reply:
Oh, yes, I did not see the explanation:

```
 * ``conan graph info xxxx --format=json``
 * ``conan create xxxx --format=json``
 * ``conan install xxxx --format=json``
 * ``conan export-pkg xxxx --format=json``
```